File-Formats/DocIO/Working-with-Word-document.md

```diff
@@ -1432,6 +1432,209 @@ using (WordDocument document = new WordDocument(assembly.GetManifestResourceStre
+        //https://help.syncfusion.com/file-formats/docio/create-word-document-in-xamarin#helper-files-for-xamarin
+    }
+}
+
+{% endhighlight %}
+
+{% endtabs %}  
+
+You can download a complete working sample from [GitHub]
+
+## Remove background in Word document
+
+You can remove background colors and images in an existing Word document by setting `NoBackground` as background type.
+
+The following code example shows how to remove background in Word document.
+
+{% tabs %}
+
+{% highlight c# tabtitle="C#" %}
+//Load Word document.
+using (WordDocument document = new WordDocument(“Input.docx” FormatType.Docx))
+{
+//Remove the existing background in Word document.
+document.Background.Type = BackgroundType.NoBackground;
+//Save the Word document.
+document.Save(“Sample.docx”), FormatType.Docx);
+}
+
+{% endhighlight %}
+
+{% highlight vb.net tabtitle="VB.NET" %}
+'Load Word document.
+Using document As WordDocument = New       WordDocument(“Input.docx"), FormatType.Docx)
+'Remove the existing background in Word document.
+document.Background.Type = BackgroundType.NoBackground;
+'Save the Word document.
+document.Save(“Sample.docx"), FormatType.Docx)
+End Using
+
+{% endhighlight %}
+
+{% highlight c# tabtitle="UWP" %}
+//"App" is the class of Portable project.
+Assembly assembly = typeof(App).GetTypeInfo().Assembly;
+//Open the file as Stream.
+using (Stream docStream = typeof(App).GetTypeInfo().Assembly.GetManifestResourceStream("Sample.Assets.Input.docx"))
+{
+    //Load file stream into Word document.
+    using (WordDocument document = new WordDocument(docStream, FormatType.Docx))
+    {
+        //Remove the existing background in Word document.
+        document.Background.Type = BackgroundType.NoBackground;
+        //Save the Word document to MemoryStream.
+        MemoryStream stream = new MemoryStream();
+        await document.SaveAsync(stream, FormatType.Docx);
+        //Save the stream as Word document file in local machine.
+        Save(stream, "Sample.docx");
+        //Please refer the below link to save Word document in UWP platform
+        //https://help.syncfusion.com/file-formats/docio/create-word-document-in-uwp#save-word-document-in-uwp
+    }
+} 
+
+{% endhighlight %}
+
+{% highlight c# tabtitle="ASP.NET Core" %}
+//Open the file as Stream.
+using (FileStream docStream = new FileStream("Input.docx", FileMode.Open, FileAccess.Read))
+{
+    //Load file stream into Word document.
+    using (WordDocument document = new WordDocument(docStream, FormatType.Docx))
+    {
+        //Remove the existing background in Word document.
+        document.Background.Type = BackgroundType.NoBackground;
+        //Save the Word document to MemoryStream.
+        MemoryStream outputStream = new MemoryStream();
+        document.Save(outputStream, FormatType.Docx);
+        outputStream.Position = 0;
+        //Download Word document in the browser.
```

Comment:
//Download the Word document in the browser.

Reply:
Done